rsyslog/init.sls

```diff
@@ -12,67 +12,6 @@ stoplogger_{{logger}}:
 {% endif %}
 
 
-{%- if grains.get('lsb_distrib_release') == "16.04" %}
-
-remove_rsyslog_8_16_0:
-  pkg.purged:
-    - name: rsyslog
-    - version: 8.16.0-1ubuntu3.1
-
-remove_rsyslog_8_2001_0:
-  pkg.purged:
-    - name: rsyslog
-    - version: 8.2001.0-1
-    - require:
-      - remove_old_omhttp
-
-remove_old_omhttp:
-  pkg.purged:
-    - name: rsyslog-omhttp
-    - version: 8.2001.0-1
-
-rsyslog:
-  pkg.installed:
-    - name: rsyslog
-    - require:
-      - remove_rsyslog_8_16_0
-      - remove_rsyslog_8_2001_0
-  file.managed:
-    - name: {{ rsyslog.config }}
-    - template: jinja
-    - source: {{ rsyslog.custom_config_template }}
-    - context:
-        config: {{ rsyslog|json }}
-
-rsyslog_mmjsonparse:
-  pkg.installed:
-    - name: rsyslog-mmjsonparse
-    - require:
-      - pkg: rsyslog
-
-{%- else %}
-rsyslog:
-  pkg.installed:
-    - name: rsyslog
-    - version: 8.2110.0-0adiscon1focal1
-    # This rsyslog version fails to install the first time and requires a manual configuration.
-    - on_fail:
-      - cmd: dpkg --configure rsyslog
-  file.managed:
-    - name: {{ rsyslog.config }}
-    - template: jinja
-    - source: {{ rsyslog.custom_config_template }}
-    - context:
-        config: {{ rsyslog|json }}
-
-rsyslog_mmjsonparse:
-  pkg.installed:
-    - name: rsyslog-mmjsonparse
```

Comment:
this package doesn't exist in unbuntu22. i think it comes for free with the package